### DISCUSSION TITLE
Show JSON parse error detail on failed theme import

### DISCUSSION THREAD

**Discussion by bczsalba** [Docile-Alligator/Infinity-For-Reddit#1033] [Ideas]
It's really hard to write any kind of programmatic theme generator because the app only shows "Parse theme failed" with no explanation. The detail is there in the code  but it gets swallowed.

**Comment by Docile-Alligator** [comment]
Ok. I will let the app show it.

**Comment by bczsalba** [comment]


**Reply by Wladefant** [reply]
I warn you, one more AFAIK or something not nice and you will be banned

**Reply by bczsalba** [reply]
What's the issue with "as far as I know"? I'm pretty sure I asked this same question on this discussion before, but either I am misremembering or it was deleted. I'm simply asking you why you closed this issue, especially since you seem to have closed most long-time inactive issues with similarly little discussion.

**Reply by Wladefant** [reply]
Haha ok, forgot the meaning of AFAIK, thought it means "why the fuck are you not fulfilling my requests" or something along the lines.

But either with that word or without your tone seems to be disrespectful and requesting. Nobody likes freeloaders who are not paying but always requesting and complaining. 

To the last question: please read the pinned discussion

**Reply by bczsalba** [reply]
I don't mean to be complaining at all, just was wondering since I saw 3 different issues closed and silenced by someone I didn't recognize as part of this repository. The only reason I'm asking about this is that @Docile-Alligator accepted the suggestion already, so even though it isn't implemented yet there isn't much to discuss in a discussion.

As a sidenote, putting all suggestions into discussions feels like a very easy way to accidentally silence 95% of new ideas, as in my experience barely anyone goes to that tab unless they have something to post. Pinning an issue that links to the pinned discussion may help avoid that.

Also, about your quote:

> Nobody likes freeloaders who are not paying but always requesting and complaining.

I feel like this is a pretty disrespectful way to frame me, a user who had an idea for a tiny change that could have improved the experience of many people. All I did was make an issue, and a year later ask why that issue was closed and silenced with no actual code referencing the idea I had.

I also maintain a fairly large (1.3k star) repository so I'm not just talking out of my ass here: treating curiosity with hostility will drive people away from your project. I already wasn't using Infinity for unrelated reasons, but after this (and the couple other threads of discussion I was involved in with You) I have practically 0 interest in trying to app out again. I had a _take it or leave it_ suggestion, asked why that suggestion was shut down by someone seemingly new to the project, (possibly) had it deleted and then when I asked a second time was told I either shut up or get banned. Not great customer experience, but it's not my userbase I am diminishing!

**Reply by Wladefant** [reply]
Thank you for your opinion.
For others reading this post, he wrote another comment that he then deleted.

As I stated above I thought the meaning of AFAIK was another so sorry for that.
I am not saying that you are a freeloader but warning you and others to not be a freeloader and write with such a tone. 

But hey everyone has different opinions, and if you have constructive idea feel free to share them (with an appropriate tone) 😄

**Reply by bczsalba** [reply]
What comment did I delete?

**Reply by Wladefant** [reply]
Some months ago on this post if I remember correctly

**Reply by bczsalba** [reply]
The content of that comment was AFAIK the same as the one at the top of this discussion, and I personally never deleted it, so I don't know what happened to it.

**Reply by Wladefant** [reply]
We will never know that 🤷‍♂️

My message to you and others: speak nice, act nice, stay topic related
